### PR TITLE
Fix Link to GPL License

### PR DESCRIPTION
**Pull request by nlordell**
Just fixes an issue with the link to the GPL license in the `COPYRIGHT.md` file.

### CODE REVIEW

File: COPYRIGHT.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ the Apache License, Version 2.0 ([`LICENSE-APACHE`](LICENSE-APACHE) or <http://w
 the MIT license, ([`LICENSE-MIT`](LICENSE-MIT) or <http://opensource.org/licenses/MIT>), at your option.
 
 Code under the following sub-directories are licensed exclusively under
-the GNU Lesser General Public License v3.0 or later ([`LICENSE`](LICENSE) or <https://www.gnu.org/licenses/lgpl-3.0-standalone.html>):
+the GNU Lesser General Public License v3.0 or later ([`LICENSE-GPL`](LICENSE-GPL) or <https://www.gnu.org/licenses/lgpl-3.0-standalone.html>):
```
Comment (MartinquaXD) [inline COPYRIGHT.md:11]: I guess you meant `LICENSE-LGPL`?

Comment (nlordell) [reply COPYRIGHT.md]: No, I had the wrong license name. Fixed.

Great eye!

Comment (MartinquaXD) [reply COPYRIGHT.md]: But then we should also be linking to `GPL` and not `LGPL`. 😅 
https://www.gnu.org/licenses/gpl-3.0-standalone.html

Comment (nlordell) [reply COPYRIGHT.md]: 🤦 🙈 😭

Comment (nlordell) [reply COPYRIGHT.md]: Should be fixed now. Thanks for the attention to detail!

**Review by MartinquaXD — APPROVED** [review]

**Pull request merged**